hnn_core/network.py

```diff
@@ -1142,6 +1143,10 @@ def add_tonic_bias(self, *, cell_type=None, amplitude, t0=0, tstop=None):
             The name of the cell type to add a tonic input. When supplied,
             a float value must be provided with the `amplitude` keyword.
             Valid inputs are those listed in  `net.cell_types`.
+        section : str | None
+            section the bias should be applied to.
```

Comment:
```suggestion
            name of cell section the bias should be applied to.
```

how will users know what are the available sections? You could list all the available options or tell the users they can look in `net.cell_types['L2_pyramidal'].sections.keys()` (check the right method) or something like that

Reply:
I followed implementation in build in cell.py line 665. Can change it but maybe we should then change both.

Reply:
build() is not really user facing but now since this parameter is user-facing, we should make sure it's clear how to provide the values. Feel free to update both.